EvaText/EvaText/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+
+namespace EvaText
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Welcome to Text Adventure University !");
+            Console.WriteLine("You're about to experience TAU, do you have what it takes?");
+
+            string response1 = Console.ReadLine();
+
+            if (response1 == "Yes")
+            {
+                Console.WriteLine("First we must stop by the Student Center.");
+                Console.WriteLine("Here we have to check your admissions status, and your financial status");
+                Console.WriteLine();
+                Console.WriteLine("Are you ready to enter The Student Center?");
+
+                string response2 = Console.ReadLine();
+                if (response2 == "Yes")
+                {
+                    //Show Student Hill Center building
+
+                    Console.WriteLine("Admissions is your first stop here at the Student Center");
+                    Console.WriteLine("");
+                    Console.Clear();
```

Comment:
Doesn't this remove the text from line 26 before the user has a chance to see it?